numpy/lib/_histograms_impl.py

```diff
@@ -453,6 +464,37 @@ def _get_bin_edges(a, bins, range, weights):
         return bin_edges, None
 
 
+def _get_geomspace_edges(n_unequal_bins, a):
```

Comment:
```suggestion
def _get_geomspace_edges(a, n_unequal_bins):
```
(keep the same order as `_get_bin_edges`)